bin/pre-commit-hook.js

```diff
@@ -44,9 +44,9 @@ function getPathForCommand( command ) {
 	const composerBinDir = path.join( __dirname, '..', 'vendor', 'bin' );
 	let path_to_command;
 	try {
-		path_to_command = execSync( 'command -v ' + command, { encoding: 'utf8' } );
-	} catch ( e ) {
 		path_to_command = path.join( composerBinDir, command );
+	} catch ( e ) {
+		path_to_command = execSync( 'command -v ' + command, { encoding: 'utf8' } );
 	}
```

Comment:
This was causing issues runing phpcbf on commits locally. I had phpcbf required globally. The existing code uses your global install by default then goes to the local install.

I think that's backwards. We have specific versions and guidelines set up in our composer.json file. Those should be followed by default, and then we can revert to the global install if needed.

Reply:
#44628 from @ockham will fix this